handler.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+const AWS = require('aws-sdk');
+const ccxt = require('ccxt');
+const axios = require('axios');
+
+module.exports.endpoint = async (event) => {
+  const params = JSON.parse(event.body);
```

Comment:
Will it work with SNS ? Because we want to activate this function using SNS event.